app/assets/javascripts/oxalis/view/action-bar/tracing_actions_view.js

```diff
@@ -312,13 +312,16 @@ class TracingActionsView extends React.PureComponent<Props, State> {
         Share
       </Menu.Item>,
     );
-    modals.push(
-      <ShareModalView
-        key="share-modal"
-        isVisible={this.state.isShareModalOpen}
-        onOk={this.handleShareClose}
-      />,
-    );
+    // This modal fetches data for the current dataset which is wrong if it is rendered too early
```

Comment:
Hmm, I think the idiomatic solution would be to listen for prop changes within the ShareModalView. That way, this error case is also covered in case the same component is used somewhere else / its usage is refactored.